nservicebus/msmq/transportconfig.md

```diff
@@ -61,7 +66,7 @@ From V4 onwards the configuration for this mechanism is implemented in the `Tran
  * `MaxRetries` (default: `5`) defines how many times a message is tried to be processed before is is moved to the *error queue* or passed to the [Second-Level Retries, SLR](/nservicebus/errors/automatic-retries.md) mechanism.
  * `ErrorQueue` (default: `error`) sets the name of the queue where poison messages are sent to (including messages that failed *MaxRerties* number of times with SLR disabled and messages which cannot be processed at all, e.g. having unparsable or missing headers)
 
-In V3 some of these setting were available via `MsqmTransportConfig` section with following 
+In V3 some of these setting were available via `MsqmTransportConfig` section with following
 
  * In V3 the `ErrorQueue` (the queue where messages that fail a configured number of times) settings can be set both via the new `MessageForwardingInCaseOfFaultConfig ` section and the old `MsmqTransportConfig` section.
- * In V3 the `MaxRetries` as well as the throttling  (`NumberOfWorkerThreads`) settings can be set only via `MsmqTransportConfig` section.
\ No newline at end of file
+ * In V3 the `MaxRetries` as well as the throttling  (`NumberOfWorkerThreads`) settings can be set only via `MsmqTransportConfig` section.
```

Comment:
@danielmarbach my editor (Atom) must have trimmed the trailing whitespace.
I don't see the harm in keeping them, but I'm not sure html generation will agree. I'll try reverting in the next commit.